### ISSUE TITLE
Forward error correction?

### ISSUE THREAD

**Issue by Codeusa** [w3c/webrtc-quic#25]
One of the lacking features of WebRTC data channels is that there is no native FEC under the hood, forcing developers to use JS or Web Assembly based alternatives. If FEC could be enabled and negotiated by both sides, this would allow for high-performance real-time data applications on more unreliable networks without retransmission.

**Comment by aboba** [comment]
@codeusa I don't believe that the IETF QUIC WG has a specification for QUIC FEC, but if there was one, this could potentially be supported by passing a dictionary of parameters as an argument to createStream(RTCQuicStreamParameters parameters).  For example: 

`dictionary RTCQuicStreamParameters {
    unsigned long   maxPacketLifetime;  // For unreliable QUIC streams
    unsigned long   maxRetransmits;  // For unreliable QUIC streams
    boolean         fec = false;  // To support forward error correction
};`

**Comment by vr000m** [comment]
I think QUIC was considering (Google's early implementations even implemented this) FEC for the tail loss in the reliable mode.

**Comment by vr000m** [comment]
I don’t think it was documented as an IETF draft. However I remember reading this.

Maybe Ian Swett can shed some light on this. 

https://docs.google.com/document/d/1Hg1SaLEl6T4rEU9j-isovCo8VEjjnuCPTcLNJewj7Nk
